scripts/integration-tests/utils/local-registry.sh

```diff
@@ -7,10 +7,15 @@ default_verdaccio_package=verdaccio@~4.11.1
 
 function startLocalRegistry {
   # Start local registry
-  tmp_registry_log=`mktemp`
+  tmp_registry_log=$(mktemp)
   echo "Registry output file: $tmp_registry_log"
   (cd && nohup npx verdaccio@~5.1.2 -c $1 &>$tmp_registry_log &)
-  YARN_IGNORE_PATH=1 yarn global add verdaccio-memory@~10.0.0
+  if [[ $(YARN_IGNORE_PATH=1 yarn --version) == "1"* ]]; then
+    YARN_IGNORE_PATH=1 yarn global add verdaccio-memory@~10.0.0
+  else
+    yarn dlx verdaccio-memory@~10.0.0
```

Comment:
Isn't this just like `npx`? i.e. it doesn't make the package globally available; it just executes it.

Reply:
Yes, the global install was removed in yarn3.

Reply:
@nicolo-ribaudo Correct, in this case it will download `verdaccio-memory` and look for a binary with the same name, it wont be able to locate one and throw.